tempto-core/src/main/java/com/teradata/tempto/internal/fulfillment/table/TableNameGenerator.java

```diff
@@ -31,8 +34,8 @@ public boolean isMutableTableName(String tableNameInDatabase)
         return tableNameInDatabase.startsWith(MUTABLE_TABLE_NAME_PREFIX);
     }
 
-    private String randomUUID()
+    private String randomSuffix()
     {
-        return UUID.randomUUID().toString();
+        return Integer.toString(abs(random.nextInt()), Character.MAX_RADIX);
```

Comment:
Still can be (unlikely) negative.

Instead of hand-crafting, consider using  `RandomUtils.randomAlphanumeric(..)` from apache commons ?